### ISSUE TITLE
wml_contrib/tkwmd still refers to now gone wmu man page

### ISSUE THREAD

**Issue by xtaran** [thewml/website-meta-language#34]
`wml_contrib/tkwmd` still refers to now gone `wmu` man page.

Probably needs to be checked for reference to other legacy parts of the WML suite.

**Comment by shlomif** [comment]
Fixed in git master. I should note that i feel that the last three issues you filed dealt with quite unimportant problems.

**Comment by xtaran** [comment]
> I should note that i feel that the last three issues you filed dealt with quite unimportant problems.

Definitely, but Github doesn't have any kind of severity for issues, otherwise I'd rated them as "minor" or so. But they still were things I'd like to see fixed upstream as I had to fix them in the Debian package (well, not this one, but the other two), and I don't want to carry these patches forever, so I filed these issues, even if they were rather unimportant.

(And yes, I just had a look at the `wml_contrib` directory closer which I didn't seem to have done since 2.0.12 or so. Was prompted by a bug report about a missing dependency against the Debian package. After that bug report I started to cross-check all `use` outside `wml_include/TheWML/` and that way stumbled over `use Tk;` in `wml_contrib/tkwmd`.)
